src/not-found-page/messages.js

```diff
@@ -0,0 +1,11 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
+
+const messages = defineMessages({
+  title: {
+    id: 'notFoundPage.title',
```

Comment:
[suggestion/code-health]: Let's try to follow the naming of translation id that exists in [Authoring MFE](https://github.com/openedx/frontend-app-authoring/blob/master/src/course-unit/messages.js#L5). 
1. Pattern - `<mfe_name>.<page_name>.<...other>` (e.g. `course-authoring.course-unit.general.alert.error.description`)
2. Kebab case

```suggestion
    id: 'category.not-found-page.title'
```

Reply:
Fixed